### ISSUE TITLE
https://nim-lang.org/choosenim/init.sh is outdated

### ISSUE THREAD

**Issue by SpotlightKid** [nim-lang/choosenim#49]
https://nim-lang.org/choosenim/init.sh needs to be updated to the latest version in the repo. It is missing the latest changes in [commit ac77dc5 "supports arm64 on macOS" ](https://github.com/nim-lang/choosenim/commit/fc240acbcfaff5fe8f3eda4bcf5fbafe84a86add).

This means that on macOS arm64 it still downloads the amd64 choosenim binary.

See also: https://github.com/dom96/choosenim/issues/347

**Comment by ringabout** [comment]
Well, it has been linked to https://github.com/nim-lang/choosenim/blob/master/scripts/choosenim-unix-init.sh in the server

**Comment by SpotlightKid** [comment]
When I download the file (with `curl -O`) , I still get the version, which has this on line 127ff.:

```sh
      if [ "$HOSTTYPE" = "arm64" ] ; then
        # TODO: We don't have arm binaries for choosenim so far, rely on Rosetta.
        local ucpu="amd64"
      fi
```

which is the previous version.

**Comment by ringabout** [comment]
@SpotlightKid Sorry for the inconvenience. We have had fixed it on the server. Does it work for you now?

**Comment by SpotlightKid** [comment]
Yes, an up-to-date `init.sh` is downloaded now. I don't have macOS so I can't test whether it downloads the right binary for macOS arm64 now, but it should.

**Comment by ringabout** [comment]
https://github.com/nim-lang/choosenim/pull/55 will be added to ensure it works on macosx_arm
